DuckDuckGo/NetworkProtection/AppTargets/BothAppTargets/NetworkProtectionTunnelController.swift

```diff
@@ -618,6 +614,8 @@ final class NetworkProtectionTunnelController: TunnelController, TunnelSessionPr
         options[NetworkProtectionOptionKey.selectedEnvironment] = settings.selectedEnvironment.rawValue as NSString
         options[NetworkProtectionOptionKey.selectedServer] = settings.selectedServer.stringValue as? NSString
 
+        options[NetworkProtectionOptionKey.excludeLocalNetworks] = NSNumber(value: settings.excludeLocalNetworks)
```

Comment:
We should really switch to passing all of `VPNSettings` encoded, but it felt like a risky change in the scope of my work here.

Reply:
Agreed, I'm fine to defer that to a separate PR, especially since this change isn't really adding much to the tech debt.